src/store/pnp.js

```diff
@@ -92,9 +92,15 @@ const mutations = {
     state.peer = peer
   },
   [PEER_DISCONNECTED] (state) {
-    state.peerConnection = undefined
-    state.peerConnectionStatus = PEER_DISCONNECTED
+    try {
+      state.peer.disconnect()
+    } catch (e) {
+      console.log(`Error disconnecting peer: ${e}`)
```

Comment:
@vickywane let's log this at error level instead of debug. `console.error`. We need to understand when `disconnect()` fails and fix it.